### DISCUSSION TITLE
Is it possible to have netdata without the visualization and dashboard built into to the agent

### DISCUSSION THREAD

**Discussion by sirdatta** [netdata/netdata#17147] [Q&A] [answered]
If we sign up for the business pro, would it be possible to have netadata agent without visualization and dashboard code. We can disable it, but the ask is, to completely drop the code for those two.

**Comment by sirdatta** [comment]
It looks like it is not possible.

**Reply by ilyam8** [reply, marked as answer]
@sirdatta, hey. You can [disable local web server](https://learn.netdata.cloud/docs/netdata-agent/securing-netdata-agents#disable-the-local-dashboard). An Agent w/o UI code - we don't produce such builds.
